workflow-examples/run_kubeapi.sh

```diff
@@ -0,0 +1,243 @@
+                                                "value": "get"
+                                              },
+                                  {
+                                    "key": "work-ctx-key",
+                                    "value": "thekey"
+                                  }
+            ]
+          }
+        ]
+      }' | jq -r '.workFlowExecutionId')"
+}
+
+run_create() {
+  EXECUTION_ID="$(curl -X 'POST' -s \
+      "${TARGET_URL}/api/v1/workflows/" \
+      -H 'accept: */*' \
+      -H 'Authorization: Basic dGVzdDp0ZXN0' \
+      -H 'Content-Type: application/json' \
+      -d '{
+            "projectId": "'$PROJECT_ID'",
+        "workFlowName": "'$WORKFLOW_NAME'",
+        "works": [
+          {
+            "workName": "kubeapiTask",
+            "arguments": [
+              {
+                    "key": "api-group",
+                    "value": ""
+                  },
+                  {
+                    "key": "api-version",
+                    "value": "v1"
+                  },
+                  {
+                    "key": "kind-plural-name",
+                    "value": "configmaps"
+                  },
+                {
+                  "key": "operation",
+                  "value": "create"
+                },
+                {
+                  "key": "resource-json",
+                  "value": "{\"apiVersion\": \"v1\",\"data\": {\"a\": \"vala\",\"b\": \"valb\"},\"kind\": \"ConfigMap\",\"metadata\": {\"name\": \"my-cm-create\",\"namespace\": \"flpath144\"}}"
+                }
+            ]
+          }
+        ]
+      }' | jq -r '.workFlowExecutionId')"
+}
+
+run_update() {
+  EXECUTION_ID="$(curl -X 'POST' -s \
+      "${TARGET_URL}/api/v1/workflows/" \
+      -H 'accept: */*' \
+      -H 'Authorization: Basic dGVzdDp0ZXN0' \
+      -H 'Content-Type: application/json' \
+      -d '{
+            "projectId": "'$PROJECT_ID'",
+        "workFlowName": "'$WORKFLOW_NAME'",
+        "works": [
+          {
+            "workName": "kubeapiTask",
+            "arguments": [
+              {
+                    "key": "api-group",
+                    "value": ""
+                  },
+                  {
+                    "key": "api-version",
+                    "value": "v1"
+                  },
+                  {
+                    "key": "kind-plural-name",
+                    "value": "configmaps"
+                  },
+                {
+                  "key": "operation",
+                  "value": "update"
+                },
+                {
+                  "key": "resource-json",
+                  "value": "{\"apiVersion\": \"v1\",\"data\": {\"a\": \"valb\",\"b\": \"vala\"},\"kind\": \"ConfigMap\",\"metadata\": {\"name\": \"my-cm-create\",\"namespace\": \"flpath144\"}}"
+                }
+            ]
+          }
+        ]
+      }' | jq -r '.workFlowExecutionId')"
+}
+
+run_flow() {
```

Comment:
what do you think about using the sdk? we are moving away from bash based tests

Reply:
not in this PR. We can open a separate story for that

Reply:
BTW piotr, this is not a test. it is an example.

Reply:
Then, add the test in a different commit with the test.

Reply:
we are moving away from bash script. please follow the new way

Reply:
yes, but I have a PR already. I don't want to rebase all the time. opened a story https://issues.redhat.com/browse/FLPATH-337 in the epic I am working on

Reply:
BTW, has anyone used it for the examples?

Reply:
removed the bash script